### PR TITLE
Fix bug where undefined is shown on UI for estimatedMins in case of ingestion data missing

### PR DESCRIPTION
**Pull request by yizheliu-amazon**
*Issue #, if available:*

*Description of changes:*
Fix bug where undefined is shown on UI for estimatedMins in case of ingestion data missing

**Before**
Screen Shot 2020-09-03 at 1 26 39 PM
Screen Shot 2020-09-03 at 1 27 34 PM

**After**
Screen Shot 2020-09-03 at 1 15 33 PM
Screen Shot 2020-09-03 at 1 16 26 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yizheliu-amazon** [comment]
will cherry-pick it to 1.10, 1.9

File: public/pages/DetectorResults/containers/AnomalyResults.tsx [hunk 1]
```diff
@@ -313,7 +313,10 @@ export function AnomalyResults(props: AnomalyResultsProps) {
   };
 
   const getInitProgressMessage = () => {
-    return detector && isDetectorInitializing && detector.initProgress
+    return detector &&
+      isDetectorInitializing &&
+      detector.initProgress &&
```
Comment (ylwu-amzn) [inline public/pages/DetectorResults/containers/AnomalyResults.tsx:318]: Use lodash get ?

**Comment by ohltyler** [comment]
Small: can you manually add to 1.10 release notes?

File: public/pages/DetectorResults/containers/AnomalyResults.tsx (hunk 1 above)
Comment (yizheliu-amazon) [reply public/pages/DetectorResults/containers/AnomalyResults.tsx:318]: will use it

**Comment by yizheliu-amazon** [comment]
> Small: can you manually add to 1.10 release notes?

good point. will add it.

File: public/pages/DetectorResults/containers/AnomalyResults.tsx (hunk 1 above)
Comment (ohltyler) [reply public/pages/DetectorResults/containers/AnomalyResults.tsx:318]: good idea, probably many places within this page and other places in the plugin where we should change to this to make it simpler

**Review by ohltyler — APPROVED** [review]
Thanks for catching this case!

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**